app/controllers/stories_controller.rb

```diff
@@ -423,7 +423,7 @@ def set_organization_json_ld
       "url": URL.organization(@organization),
       "image": ProfileImage.new(@organization).get(width: 320),
       "name": @organization.name,
-      "description": @user.summary.presence || "404 bio not found"
+      "description": @organization.summary.presence || "404 bio not found"
```

Comment:
I'm pretty sure this should be `@organization` and not `@user` like it was prior to this commit... 🤔